application_sandbox/overlapping_frames/main.cpp

```diff
@@ -93,7 +100,7 @@ uint32_t vert[] =
 uint32_t frag[] =
 #include "triangle.frag.spv"
     ;
-};
+};  // namespace gBuffer
```

Comment:
This looks like a bad clang-format.

Reply:
Or at least it should not have the ; on the end.

Reply:
done